client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -111,6 +129,16 @@ class TaxonomyManagerListItem extends Component {
 		return decodeEntities( term.name ) || translate( 'Untitled' );
 	};
 
+	viewTaxonomyPosts = () => {
```

Comment:
Shouldn't we move this callback to the `ViewTaxonomyMenuItem` Component instead, It seems (at least to me :P) reasonable to say that if we want to preview a link, we only always have the same callback, don't you think?